src/main/java/com/f1soft/bachaat/repository/UserRepository.java

```diff
@@ -1,10 +1,14 @@
 package com.f1soft.bachaat.repository;
 
 import com.f1soft.bachaat.entity.User;
+import org.springframework.data.domain.Example;
+import org.springframework.data.domain.Page;
```

Comment:
this is an unnecessary file. I don't see any use of this file in this pr

Reply:
done